tests/bluetooth/tester/src/btp/btp_aics.h

```diff
@@ -0,0 +1,26 @@
+/* aics.h - Bluetooth tester headers */
```

Comment:
```suggestion
/* btp_aics.h - Bluetooth tester headers */
```

Applies to several files in this PR

Reply:
Done